app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,26 @@ def show
     @post = @user.posts.find(params[:id])
     @comments = @post.comments
   end
+
+  def create
+    @post = current_user.posts.new(post_params)
+
+    if @post.save
+      redirect_to user_posts_path(current_user)
+
+    else
+      flash[:alert] = 'Something went wrong'
+      render 'new'
+    end
+  end
```

Comment:
- [ ] on creating a post I encounter an error, kindly fix that, 
![Screenshot from 2023-08-26 01-18-00](https://github.com/Jrmcuervo/Blog-app_ror/assets/66097371/087d08b7-2ba4-4e51-9b7a-0ee03186ca8b)